<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>p6_chatBDAS</class>
 <widget class="QWidget" name="p6_chatBDAS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>501</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>chatBDAS</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>424</y>
     <width>741</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_search">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>420</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="pics.qrc">
     <normaloff>:/images/data/发送-实色.png</normaloff>:/images/data/发送-实色.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QTextBrowser" name="p6_textBrowser_user">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>631</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="p6_textBrowser_bdas">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>631</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="p6_icon_user">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>70</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pics.qrc">:/images/data/用户 (1).png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="p6_icon_bdas">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pics.qrc">:/images/data/BDAS.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="p6_pushButton_exit">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>10</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>EXIT</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="pics.qrc"/>
 </resources>
 <connections/>
</ui>
